### DISCUSSION TITLE
openvino without igpu?

### DISCUSSION THREAD

**Discussion by sunnysingh-84** [blakeblackshear/frigate#5445] [General]
Hi,

Am i correct in understanding that you can use Openvino without an igpu?

**Comment by NickM-27** [comment]
yes, openvino can also be used with the NCS2 NPU or with an intel CPU

**Reply by sunnysingh-84** [reply]
excellent so essentially three or four options?

openvino:
a) intel cpu
b) intel cpu + 6th gen or newer igpu
c) intel cpu + ncs2 npu
d intel cpu +  6th gen or newer igpu + ncs2 npu

**Reply by NickM-27** [reply]
I don't believe you can run NCS2 along with other detectors
